src/stream-tcp.c

```diff
@@ -5763,10 +5920,49 @@ TmEcode StreamTcpThreadInit(ThreadVars *tv, void *initdata, void **data)
     stt->counter_tcp_ssn_memcap = StatsRegisterCounter("tcp.ssn_memcap_drop", tv);
     stt->counter_tcp_ssn_from_cache = StatsRegisterCounter("tcp.ssn_from_cache", tv);
     stt->counter_tcp_ssn_from_pool = StatsRegisterCounter("tcp.ssn_from_pool", tv);
+    if (stream_config.ssn_memcap_policy != EXCEPTION_POLICY_NOT_SET) {
```

Comment:
Noticing this same / similar looking loop in many places. Can we use a helper func for it?